en/gps_compass/rtk_gps.md

```diff
@@ -22,7 +22,8 @@ It also highlights devices that connect via the CAN bus, and those which support
 
 Device | GPS | Compass | [DroneCAN](../dronecan/index.md) | [GPS Yaw](#configuring-gps-as-yaw-heading-source) | PPK
 :--- | :---: | :---:  | :---:  | :---:  | :---:
-[ARK RTK GPS](https://arkelectron.com/product/ark-rtk-gps/) | F9P  | ICM42688p | &check; | [Dual F9P][DualF9P] |
+[ARK RTK GPS](../dronecan/ark_rtk_gps.md) | F9P  | BMM150 | &check; | [Dual F9P][DualF9P] |
+[ARK MOSAIC-X5 RTK GPS](../dronecan/ark_mosaic__rtk_gps.md) | Mosaic-X5  | IIS2MDC | &check; | [Septentrio Dual Antenna][SeptDualAnt] |
```

Comment:
@patrikpordi Is this really Septentrio Dual Antenna ? If so, perhaps link to that information from the ARK MOSAIC-X5 page.

Reply:
Ignore that. First thing you said!!!